11_register_hosts.sh

```diff
@@ -40,6 +40,12 @@ function make_bm_workers() {
     done
 }
 
+if [ "$TEST_CUSTOM_MAO" = true ]; then
+  # Now that the deployment is up, replace the machine-api-operator image with
+  # one that needs to be tested in this CI before we bring up the workers.
+  ./run-custom-mao.sh quay.io/sdasu/machine-api-operator:sdasu-fix sadasu sdasu-fix
```

Comment:
This should use `${SCRIPTDIR}` instead of a relative path, and also you need to remove the now-optional arguments here I think?